packages/@react-spectrum/actiongroup/test/ActionGroup.test.js

```diff
@@ -353,34 +363,6 @@ describe('ActionGroup', function () {
     expect(button2).toHaveAttribute('aria-checked', 'false');
   });
 
-  it('ActionGroup supports shift + arrow keys to extend selection', function () {
```

Comment:
Yeah, this doesn't seem super expected for an ActionGroup anyway. We kinda just got it because we used the selection hooks. I don't think it's worth reimplementing...